coalib/bears/Bear.py

```diff
@@ -9,6 +9,12 @@
 from coalib.settings.ConfigurationGathering import get_config_directory
 
 
+class classproperty(property):
+
+    def __get__(self, obj, type_):
+        return self.fget.__get__(None, type_)()
```

Comment:
Also such a function would belong to the module `coalib.misc.Decorators` and should go into another commit + according tests^^